docs/Encoder.md

```diff
@@ -1,57 +1,103 @@
+There are two design requirements that must be met for the encoder subsystem of the AMDC:
+
+1. Interface standard differential incremental encoders (5V differential unipolar signals) to FPGA.
+
+2. Provide a high-bandwidth front-end so that the encoder line count is limited by the FPGA speed, not circuitry.
+
 
 ## Block Diagram
 
 <img src="images/amdc-encoder.svg" />
 
 ### 1. Encoders
-Two encoders can be connected to AMDC Rev D simultaneously. These encoders should be incremental encoders outputting differential ABZ signals. For each pulse, there is non-inverting and inverting output. For example, for pulse A, there is a non-inverting output A and an inverting output A̅. If the encoder gives single-ended ABZ signals, an adapter which converts from single-ended to differential should be used to connect to AMDC. Encoder ground should be connected to the AMDC ground through a port of d-sub connector. Encoder can be supplied from a separate source or from AMDC which gives 5 V supply through a port of d-sub connector.
-### 2. D-Sub Connector
-Each d-sub connecter has 9 ports – 6 for differential ABZ signals, 2 for ground signals and 1 for 5 V supply which can be connected to encoder as a DC supply. A jumper should be connected if encoder is powered by 5 V from AMDC. More information on d-sub connector can be found [here]( https://www.alliedelec.com/m/d/c3366066d9274ddf3c20bc3008518f5b.pdf?src-supplier=Allied+Electronics)
-### 3. Differential to single-ended converters
-After passing through d-sub connector, differential ABZ signals (A, A̅, A, B̅, Z, Z̅) are converted to single-ended ABZ signals (OUT A, OUT B, OUT Z) using [AM26C32](http://www.ti.com/lit/ds/symlink/am26c32.pdf) chip. This chip is supplied by Vcc = 5 V DC source from AMDC. The following table summarizes absolute maximum ratings taken from the datasheet:   
 
-| Parameter | MIN            | MAX                | 
-|-------------------|-----------------|----------------------|
-| Supply voltage Vcc             |                 | 7 V          |
-| Input voltage (A, A̅, A, B̅, Z or Z̅)           | -11 V            | 14 V     |
-| Differential input voltage      | -14 V            | 14 V    |
-| Output voltage (OUT A, OUT B, OUT Z) | -0.5 V              | (Vcc + 0.5) V  |
+The AMDC REV D hardware supports two independent encoder interfaces. These must be incremental encoders outputting differential ABZ signals (i.e. the encoder outputs both the non-inverting and inverting signal, e.g. A and A̅). If the encoder only provides single-ended ABZ outputs, an adapter must be used between the encoder and the AMDC which converts the single-ended outputs to the differential signals required by the AMDC. For proper operation, the encoder ground should be connected to the AMDC ground via a pin on the DB9 connector. For powering the encoder, the user has two options: the AMDC provides 5V power via the DB9 connector, or the user can use an external power supply.

```

Comment:
... yes... kind of... The encoder interface will probably work just fine without connecting to a shared common, since it is differential. But, of course, the user SHOULD connect to shared common.

The reason the sentence is there is to make sure they do, since it might intermittently work without the GND connection (e.g. it works without PWM EMI, then doesn't when PWM is on)

We can remove it if you think it actually is obvious to connect the common GND!

Reply:
Thanks for explaining. Let's keep the sentence.